### PR TITLE
docs: add commands to trigger ci doc

### PR DESCRIPTION
**Pull request by purelind**
### What problem does this PR solve?

Add doc of commands to communicate with ci pipeline.

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

How it Works:

### Check List 

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note 

```release-note
None
```

### CODE REVIEW

File: CI.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+   * Yes，this ci will be triggered only when your comment on pr matched trigger phrase.
+
+   * No，this ci will be triggered by every new commit on current pr，comment matched trigger phrase also trigger ci pipeline.
+
+## Commands
+
+| ci pipeline                              | Command                                                      | Only use trigger phrase for build triggering |
+| ---------------------------------------- | ------------------------------------------------------------ | -------------------------------------------- |
+| tidb_ghpr_build                          | .\*\/(merge\|run(-all-tests\|-build).\*)                     | No                                           |
```
Comment (kennytm) [inline CI.md:19]: why use regular expression 😂 

it is not so different from listing out all commands: 

> /merge<br>/run-all-tests<br>/run-build

but the regexp here needs more effort to comprehend.

File: CI.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+1. ci pipeline will be triggered when your comment on pull request matched trigger phrase.
+
+2. Some ci pipelines only response to people belong to specific organization like pingcap & tikv.
+
+3. "**Only use trigger phrase for build triggering**". What does that mean?
+
+   * Yes，this ci will be triggered only when your comment on pr matched trigger phrase.
+
+   * No，this ci will be triggered by every new commit on current pr，comment matched trigger phrase also trigger ci pipeline.
```
Comment (kennytm) [inline CI.md:13]:
```suggestion
   * Yes, this ci will be triggered only when your comment on pr matched trigger phrase.

   * No, this ci will be triggered by every new commit on current pr, comment matched trigger phrase also trigger ci pipeline.
```

File: CI.md (hunk 1 above)
Comment (purelind) [reply CI.md:19]: Emm.  Listing out all commands is more simple and clear. I will correct them later.

File: CI.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+1. ci pipeline will be triggered when your comment on pull request matched trigger phrase.
+
+2. Some ci pipelines only response to people belong to specific organization like pingcap & tikv.
+
+3. "**Only use trigger phrase for build triggering**". What does that mean?
+
+   * Yes, this ci will be triggered only when your comment on pr matched trigger phrase.
+
+   * No, this ci will be triggered by every new commit on current pr, comment matched trigger phrase also trigger ci pipeline.
```
Comment (tisonkun) [inline CI.md:13]:
```suggestion
1. ci pipeline will be triggered when your comment on pull request matched trigger phrase.
2. Some ci pipelines only response to people belong to specific organization like pingcap & tikv.
3. "**Only use trigger phrase for build triggering**". What does that mean?
   * Yes, this ci will be triggered only when your comment on pr matched trigger phrase.
   * No, this ci will be triggered by every new commit on current pr, comment matched trigger phrase also trigger ci pipeline.
```

also suggest use other expression such as "Only triggered by command" which may be easy to understand.

**Review by kennytm — APPROVED** [review]

File: CI.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Commands to trigger ci pipeline
+
+## Guide
+
+1. ci pipeline will be triggered when your comment on pull request matched command.
+2. Some ci pipelines only response to people belong to specific organization like pingcap & tikv.
```
Comment (tisonkun) [inline CI.md:6]: I think we're in pingcap/tidb so you can just talk about this repo itself.

BTW, what about consistently use "CI" in content and rename filename to "ci.md"?

**Review by tisonkun — APPROVED** [review]
LGTM.

**Review by zhouqiang-cl — APPROVED** [review]
LGTM

**Pull request merged**